aiida/cmdline/commands/__init__.py

```diff
@@ -25,6 +25,8 @@
 )
 
 # Import to populate the `verdi data` sub commands
-from aiida.cmdline.commands.cmd_data import (
-    cmd_array, cmd_bands, cmd_cif, cmd_dict, cmd_remote, cmd_structure, cmd_trajectory, cmd_upf
-)
+## Note: this does not seem to be necessary. With this, running
```

Comment:
If it is not necessary let's remove it and not keep dead code

Reply:
OK, I was just checking this. Indeed, the reason is that they are defined as entry points.
The latest test fails because `remote` was not in the entry points. I've fixed this now.